apps/scan/backend/src/store.ts

```diff
@@ -565,14 +561,6 @@ export class Store {
       return true;
     }
 
-    if (
-      isFeatureFlagEnabled(
-        BooleanEnvironmentVariableName.ENABLE_CONTINUOUS_EXPORT
-      )
-    ) {
-      return true;
-    }
-
```

Comment:
This is also related to scanner backups. tldr: You have to export a backup before you can unconfigure a scanner. Continuous export should render this moot (because you're essentially always backed up), but I want to think through things fully when I work on overall backup functionality cleanup.

For now, I'm just removing the reference to the `ENABLE_CONTINUOUS_EXPORT` feature flag, which is itself removed in a subsequent commit.